h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -261,7 +262,7 @@ def mojo_predict(model, tmpdir, mojoname, glrmReconstruct=False, get_leaf_node_a
     mojoZip = os.path.join(tmpdir, mojoname) + ".zip"
     if not(zipFilePath==None):
         mojoZip = zipFilePath
-    genJarDir = str.split(str(tmpdir),'/')
+    genJarDir = str.split(os.path.realpath("__file__"),'/')
```

Comment:
what is the reason for this?

Reply:
I changed the directory to write the mojo.zip into.  Originally, it is written off h2o-3/h2o-py/tests/results directory.  This means tmpdir = /absolutepath/h2o-3/h2o-py/tests/results.  However, with @Pscheidl suggestion, I use tempfile and the directory is no longer part of h2o-3 path. 

I need to find the directory of the genmodel.jar and it is off the h2o-3/h2o-genmodel/...   Hence, I changed genJarDir from str.split(str(tmpdir), '/') to str.split(os.path.realpath("__file__"), '/') which will find the directory of the file being executed.  The current file that is being executed is located in /absolute-path/h2o-3/h2o-py/tests/testdir_algos/gam/.  Hence, we can use this to find where the genmodel.jar is located.